libtock-sync/sensors/distance.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <libtock/sensors/distance.h>
+#include <libtock/tock.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Read the distance sensor synchronously.
+//
+// ## Arguments
+//
+// - `distance`: Set to the distance value in millimeters.
+//
+// ## Return Value
+//
+// A returncode indicating whether the distance read was completed
+// successfully.
+returncode_t libtocksync_distance_read(int* distance);
```

Comment:
These function do not have an async version, I would place them only on `libtock/sensors`.